content/doc/book/using/using-agents.adoc

```diff
@@ -196,3 +196,26 @@ Building remotely on agent1 in workspace /home/jenkins/workspace/First Job to Ag
 agent1
 Finished: SUCCESS
 ----
+
+=== Launching agent from console
```

Comment:
This needs to be one level lower as a heading so that it is at the "root" level of the page.  It is a top level topic, not a subtopic of the Docker section that precedes it.
```suggestion
== Launching agent from console
```